frontend/src/index.tsx

```diff
@@ -1,2 +1,2 @@
-import 'normalize.css';
+// import 'normalize.css';
```

Comment:
Was this left in accidentally?

Reply:
I just commented it out for now, would it better to remove it?

Reply:
Probably, if fine by you..?